test/positivity.lean

```diff
@@ -14,6 +14,16 @@ import tactic.positivity
 This tactic proves goals of the form `0 ≤ a` and `0 < a`.
 -/
 
+/-  Test for instantiating meta-variables.  Reported on
+Stream: metaprogramming/tactics
+Topic: New tactic: `positivity`
+Date: Sept 25th, 2022.   -/
```

Comment:
You can probably just link to https://leanprover.zulipchat.com/#narrow/stream/239415-metaprogramming-.2F-tactics/topic/New.20tactic.3A.20.60positivity.60/near/300638974 😀

Reply:
I would, but the link exceeds the 100 characters limit...

Reply:
It's fine! The style linter ignores all lines containing `http`.

Reply:
Ah, I did not know this!

I'll wait for linting to finish, to make sure that there are no issues with the actual PR and then will push the link and the shortened test with `0 ≤ 0` instead of `0 ≤ 1`.